src/app/login/Login.tsx

```diff
@@ -4,13 +4,36 @@ import Box from '@mui/material/Box';
 import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 import { styles } from './Login.styles';
 import { useLocale } from 'hooks';
 import { LoginForm } from './loginForm/LoginForm';
+import { loginAction } from 'api/actions/auth/authActions';
+import { useMutation } from 'api/hooks';
+import { LoginPayload } from 'api/actions/auth/auth.types';
+import { useAuth } from 'hooks/useAuth/useAuth';
+import { AppRoute } from 'routing/AppRoute.enum';
 
 export const Login = () => {
   const { formatMessage } = useLocale();
+  const { mutate, data, isLoading, isError } = useMutation<LoginPayload>(loginAction);
```

Comment:
Wszystko co z api do kontenera. Pamiętasz? SRP (Single Responsible Principle). Także to do LoginContainer